releasetasks/templates/mobile/bouncer_aliases.yml.tmpl

```diff
@@ -0,0 +1,44 @@
+{% set buildername = "release-{}-{}_bouncer_aliases".format(branch, product) %}
+
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    requires:
+        - "{{ stableSlugId('publish_release_human_decision') }}"
```

Comment:
I'd rather make this dependent on push to cdns. If push to cdns fails, we shouldn't update the bouncer aliases.